src/user/tipsandtricks.rst

```diff
@@ -149,8 +149,22 @@ In order to override the default behavior, a user can set the environment variab
 
 .. note::
    
-   Note that you should select the cudatoolkit version most appropraite for your GPU; currently, we have "10.2", "11.0", "11.1", and "11.2" builds available, where the "11.2" builds are compatible with all cudatoolkits>=11.2. (At the time of writing, there seems to be a bug in how the cuda builds are resolved by mamba, defaulting to cudatoolkit==10.2; thus, it is prudent to be as explicit as possible like above or by adding ``cudatoolkit>=11.2`` or similar to the line above.) 
+   Note that you should select the cudatoolkit version most appropraite for your GPU; currently, we have "10.2", "11.0", "11.1", and "11.2" builds available, where the "11.2" builds are compatible with all cudatoolkits>=11.2. (At the time of writing, there seems to be a bug in how the cuda builds are resolved by mamba, defaulting to cudatoolkit==10.2; thus, it is prudent to be as explicit as possible like above or by adding ``cudatoolkit>=11.2`` or similar to the line above.)
 
+For context, installing TensorFlow 2.7.0 with the following command:
+
+.. code-block:: shell
+
+    CONDA_CUDA_OVERRIDE="11.2" mamba install "tensorflow==2.7.0=cuda*" -c conda-forge
+
+results in approximately 2 GB of packages to download while using this:
+
+.. code-block:: shell
+
+    CONDA_CUDA_OVERRIDE="11.2" mamba install "tensorflow=2.7.0=cpu*" -c conda-forge 
+
+results in approximately 200 MB to download. That is a significant bandwidth and storage wasted if one only needs the ``-cpu`` variant!
+=======
```

Comment:
This addition looks a little odd to me in github. Did you delete the lines below 167?

Reply:
No @kathatherine, I didn't make any changes at this place, and infact there aren't any more lines to this file after 167. I am pushing the changes after making all the suggested changes. Please do let me know if this still remains the same.

Reply:
Okay, I get it now, there was a merge conflict, may be that was the reason. I have fixed it now. :)

Reply:
Ah ha! That would make sense.